public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/appInsights.json

```diff
@@ -447,7 +447,7 @@
         "content": "<a style=\"color: #ffffff;\"href=\"https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/${sub}/resourceGroups/${rg}/providers/microsoft.insights/components/${res}/failures\" target=\"_blank\">\n<div>\n  <img src=\"https://images.vexels.com/media/users/3/153978/isolated/preview/483ef8b10a46e28d02293a31570c8c56-warning-sign-colored-stroke-icon-by-vexels.png\" width=\"40px\"/>\n  <p style=\"font-size:16px; margin-bottom:0px; margin-top:0px;\"> Failures </p>\n  <p style=\"margin-top: 0px;\">${res}</p>\n</div>\n</a>\n",
```

Comment:
sorry I know this is not related to this pr but something we noticed while looking at this yesterday is that this is not viewable on our light themes. You can test it yourself if you go to settings > preferences > and select Light and save you'll notice that these links with `color: #ffffff` I would recommend removing these and using our Link attribute for panels instead if possible

I also think the image feels a bit off to me? I keep thinking the svg is not scalling right or something but it's just sort of an off kilter design I guess? https://images.vexels.com/media/users/3/153978/isolated/preview/483ef8b10a46e28d02293a31570c8c56-warning-sign-colored-stroke-icon-by-vexels.png I suppose it's somewhat subjective but I think we're kind of better off without it?

Reply:
Thank you for pointing this out, I have gotten rid of the `color: #fffff` attribute and I have also changed the icon

I would like to keep the link+ icon combo as it is right now so that it more closely matches what we have in the azure portal. Do you like the idea of having the default link formatting + an icon that does not look like it is off?